mainapp/templates/mainapp/privaterescuecamp_form.html

```diff
@@ -3,7 +3,7 @@
 
 {% block content %}
 <h1 class="text-center">Enter New Camp Details</h1>
-<h2 class="text-center">--</h2>
+<h2 class="text-center">പുതിയ ക്യാമ്പിൽ വിശദാംശങ്ങൾ  നൽകുക </h2>
```

Comment:
It should "campinte" right?

Reply:
@biswaz yes

Reply:
@biswaz   it should be "campinte" instead of Camp or  else.
i am unable to get you

Reply:
its ക്യാമ്പിന്റെ , not ക്യാമ്പിൽ